multimedia/VLC/files/vlc

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Maybe I missed it, but it seems like the Portfile does not actually use this file at all?